bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -18,6 +18,16 @@
 - shell: git rev-parse --show-toplevel 2>/dev/null
   register: toplevel
 
+- apt:
+    name: aptitude
+    state: latest
+    update_cache: yes
+  become: true
```

Comment:
Here and below, the `become: true` is unnecessary because the whole playbook is executed with `become: true` (see `bootstrap/playbook.yml`).